mediator/fd_cesm.yaml

```diff
@@ -863,292 +969,244 @@
      - standard_name: Foxx_swnet_vdf
        alias: mean_net_sw_vis_dif_flx
        canonical_units: W m-2
-       description: ocean import - net shortwave visible diffuse radiation to ocean
+       description: med export to ocn - net shortwave visible diffuse radiation to ocean
      #
      - standard_name: Foxx_swnet_idr
        alias: mean_net_sw_ir_dir_flx
        canonical_units: W m-2
-       description: ocean import - net shortwave ir direct radiation to ocean
+       description: med export to ocn - net shortwave ir direct radiation to ocean
      #
      - standard_name: Foxx_swnet_idf
        alias: mean_net_sw_ir_dif_flx
        canonical_units: W m-2
-       description: ocean import - net shortwave ir diffuse radiation to ocean
+       description: med export to ocn - net shortwave ir diffuse radiation to ocean
      #
      - standard_name: Foxx_swnet_afracr
        canonical_units: W m-2
-       description: ocean import - net shortwave radiation times atmosphere fraction
+       description: med export to ocn - net shortwave radiation times atmosphere fraction
      #
      - standard_name: Foxx_taux
        alias: mean_zonal_moment_flx
        canonical_units: N m-2
-       description: ocean import - zonal surface stress
+       description: med export to ocn - zonal surface stress
      #
      - standard_name: Foxx_tauy
        alias: mean_merid_moment_flx
        canonical_units: N m-2
-       description: ocean import - meridional surface stress
+       description: med export to ocn - meridional surface stress
      #
      - standard_name: Fioi_swpen_ifrac_n
        alias: mean_sw_pen_to_ocn_ifrac_n
        canonical_units: W m-2
-       description: ocean import - net shortwave radiation penetrating into ice and ocean times ice fraction for thickness category 1
+       description: med export to ocn - net shortwave radiation penetrating into ice and ocean times ice fraction for thickness category 1
      #
      - standard_name: Sf_afrac
        canonical_units: 1
-       description: ocean import - fractional atmosphere coverage wrt ocean
+       description: med export to ocn - fractional atmosphere coverage wrt ocean
      #
      - standard_name: Sf_afracr
        canonical_units: 1
-       description: ocean import - fractional atmosphere coverage used in radiation computations wrt ocean
+       description: med export to ocn - fractional atmosphere coverage used in radiation computations wrt ocean
+     #
+     #-----------------------------------
+     # section: river import to med
+     #-----------------------------------
+     #
+     - standard_name: Flrr_flood
+       canonical_units: kg m-2 s-1
+       description: river import to med - water flux due to flooding
+     #
+     - standard_name: Flrr_flood_wiso
+       canonical_units: kg m-2 s-1
+       description: river import to med - water flux due to flooding for 16O, 18O, HDO
+     #
+     - standard_name: Flrr_volr
+       canonical_units: m
```

Comment:
Are these the correct units? (Ditto for all other vol variable desc.)

Reply:
@dpsarmie The fd_cesm.yaml is used by CESM only. We generally follow a policy of reviewing only common code.

Reply:
@DeniseWorthen Oh wow, my brain somehow interpreted NA as new additions, not 'not applicable', when you listed the files above. I'll exclude those and keep on with the review.